### ISSUE TITLE
Json nulls missing in response

### ISSUE THREAD

**Issue by Queatz** [arangodb/arangodb-java-driver#163]
I'm facing a bug where responses as json (using `String.class`) are not including null values in arrays.  This is causing my array indexes to be wrong

This is my query:

```
for v1 in Collection filter v1.kind == 'person' return [v1._key, v1.kind, v1.name, v1.about, (for v2 in Collection filter v2.target == v1._key and v2.kind == 'member' and v2.concluded_on == null and v2.concluded_on == null return [v2._key, v2.kind])]
```

If I run this query in the Web UI, I properly see the values that are null:

```json
[
  [
    "121680",
    "person",
    null,
    "",
    []
  ],
  [
    "3272",
    "person",
    null,
    "\"Calligraphy we'll do later.\" - Judy Yeo",
    [
      [
        "136727",
        "member"
      ],
      [
        "136708",
        "member"
      ]
    ]
  ]
]
```

However, with the following code:

```java
arangoDatabase.query(aql, vars, null, String.class)
```

I am getting this as the first response:

```json
["121680","person","",[]]
```

The `null` is missing.

When I read my response, I get an invalid index where I am expecting my array.

**Comment by mpv1989** [comment]
Thanks for reporting!

Fixed it in version 4.3.1 which I just released.

**Comment by Queatz** [comment]
Awesome, thanks!
